### ISSUE TITLE
Feature: How can I add mark tag when use generateHtmlFromNodes 

### ISSUE THREAD

**Issue by mrvegazhou** [facebook/lexical#5695]
this generateHtmlFromNodes  function return result ：
<p class="PlaygroundEditorTheme__paragraph"><span style="white-space: pre-wrap;">1111 1111 1111 </span><span style="white-space: pre-wrap;">0000</span><span style="white-space: pre-wrap;"> </span><span style="white-space: pre-wrap;">0000</span></p>
the mark tag not in html 
so , If I want to add mark tag in html , how fix it ?
<img width="425" alt="image" src="https://github.com/facebook/lexical/assets/32482520/23425192-78f8-486a-80ec-2b603d9fc60b">

**Comment by acywatson** [comment]
Ah, you'd just need to implement exportDOM on your MarkNode:

https://lexical.dev/docs/concepts/serialization#lexical---html

**Comment by mrvegazhou** [comment]
I add method exportDOM to the markNode class, 
exportDOM(): DOMExportOutput {
    const element = document.createElement('MARK');
    element.setAttribute('class', 'PlaygroundEditorTheme__mark PlaygroundEditorTheme__ltr selected');
    return {element};
  }
  but run $generateHtmlFromNodes(editor, null), I got nothing.
what's the reason？

**Comment by etrepum** [comment]
@mrvegazhou it's because of the implementation of `excludeFromCopy`,  you will need to override it to return false if you want its output to be included.

```ts
  excludeFromCopy(destination: 'clone' | 'html'): boolean {
    return destination !== 'clone';
  }
```

**Comment by PeterDekkers** [comment]
This seems to be fixed in the latest nightly of Lexical, and will be fixed once `0.24.1` is released. No need to do anything, as it's supported by the default `TextNode`.
